common/client-libs/validator-client/src/nymd/wallet/mod.rs

```diff
@@ -0,0 +1,217 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::ValidatorClientError;
+use cosmos_sdk::bip32::{DerivationPath, XPrv};
+use cosmos_sdk::crypto::secp256k1::SigningKey;
+use cosmos_sdk::crypto::PublicKey;
+use cosmos_sdk::tx::SignDoc;
+use cosmos_sdk::{tx, AccountId};
+
+pub const DEFAULT_COSMOS_DERIVATION_PATH: &str = "m/44'/118'/0'/0/0";
+// TODO: probably if this is to be made into a generic-cosmos library, the default
+// prefix should be "cosmos"
+pub const DEFAULT_BECH32_ADDRESS_PREFIX: &str = "punk";
```

Comment:
I think we have a centralized config location for these now?

Reply:
Fair point. Though I don't think `DEFAULT_COSMOS_DERIVATION_PATH` should be moved anywhere as from what I understand, it's cosmos-validator specific

Reply:
This derivation path is not tied to validators (like Tendermint) but rather any sdk account.

The purpose of using this is to maintain compatibility with ledger and golang CLI. If you do not have ledger support in this library, that is not so important (the cosmos ledger app only works for `m/44'/108'` prefix).

The other point would be that someone could import the same mnemonic in your nym rust client and in the wasmd cli tool and they would produce the same address. If this is important, it is good to stick with the default. If it is not, then there is no strong need to stick with it and you can let users configure it freely (every client may use a different one even). 

In CosmJS we allow library users to pick what they want. And it only has really been an issue for ledger compatibility.